.circleci/config.yml

```diff
@@ -81,11 +81,13 @@ jobs:
             - run: pip install .
             - run: |
                   mkdir test_preparation
-                  echo "tests" > test_preparation/test_list.txt
-                  echo "tests" > test_preparation/examples_test_list.txt
-            - run: python utils/tests_fetcher.py --filter_tests
-            - run: mv test_list.txt test_preparation/filtered_test_list.txt
-            - run: mv test_repo_utils.txt test_preparation/test_repo_utils.txt
+                  echo -n "tests" > test_preparation/test_list.txt
+                  echo -n "tests" > test_preparation/examples_test_list.txt
+            - run: |
+                  echo -n "tests" > test_list.txt
+                  python utils/tests_fetcher.py --filter_tests
+                  mv test_list.txt test_preparation/filtered_test_list.txt
+            - run: echo -n "tests/repo_utils" > test_preparation/test_repo_utils.txt
```

Comment:
`filter_tests` won't reproduce `test_repo_utils.txt` if I understand correctly. So we need to provide here manually

Reply:
Let's put this one in the previous block (so line 86)!

Reply:
done